Add alert banner to system items dialog #1608

joshdimanteto opened the pull request:
## Description

add logic to disable the banner if it is system_ids are equal i.e the item hasn't moved 
removes waits in cypress testing

## Testing instructions

Add a set up instructions describing how the reviewer should test the code

- [ ] Review code
- [ ] Check Actions build
- [ ] Review changes to test coverage
- [ ] {more steps here}

## Agile board tracking

closes #1608

joshdimanteto committed:
Add alert banner to system items dialog #1608

joshdimanteto committed:
Merge branch 'develop' into add-alert-banner-for-system-items-dialog-#1608

joelvdavies commented on src/systems/systemItemsDialog.component.tsx:
```diff
@@ -276,6 +278,21 @@ const SystemItemsDialog = React.memo((props: SystemItemsDialogProps) => {
+                    infoTooltipTitle={
+                      selectedRules && selectedRules[0]
+                        ? `The items usage statuses will be updated to ${selectedRules[0].dst_usage_status?.value}, as defined by the rules`
+                        : 'The items usage statuses we remain the same, as defined by the rules'
```
```suggestion
                        ? `The items usage statuses will be updated to ${selectedRules[0].dst_usage_status?.value}, as defined by the rules`
                        : 'The items usage statuses will remain the same, as defined by the rules'
```
The first of these can probably be the same as the items dialog one or vice versa.  `The item’s usage status will be updated to ${selectedRules?.[0]?.dst_usage_status?.value ?? selectedItem?.usage_status}, as defined by the rules`
The apostrophe used is a wierd one though, might have come from word or something, I get
<img width="1012" height="69" alt="image" src="https://github.com/user-attachments/assets/7fc88744-f145-4f6a-8e70-81a98bd1d636" />

joshdimanteto replied:
This does have to be slightly different as you can move many in systems item dialog and you can only move one in the items dialog

joelvdavies replied:
Thats fine. I think technically it would be `items'` if there are multiple but I wouldnt bother too much as it would depend on if you seleted one or many, what about the wierd character? Also the we needs to be updated as above, it doesnt make sense.

joshdimanteto committed:
address review comments #1608

update the itemsDialog alert message when item system id matches
parent system id
